src/app/(navigation)/layout.tsx

```diff
@@ -0,0 +1,16 @@
+import Navigation from "../_component/common/Navigation";
+
+export default function DashboardLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
```

Comment:
PropsWithChildren를 사용한다면 더욱 더 간결하게 작성할 수 있을거같습니다!
```suggestion
export default function DashboardLayout({ children }: PropsWithChildren<{}>) {
```

Reply:
아하 저번에 멘토님께서 말하셨던 건가..? 저는 다 까먹어버렸네요..
바로 반영하도록 하겠습니다!

Reply:
이전에 멘토님이 말씀해주신 후에 이런 포스팅을 보게되었어요!  
[PropsWithChildren는 안전한 타입일까? ](https://velog.io/@kkojae91/PropsWithChildren%EB%8A%94-%EC%95%88%EC%A0%84%ED%95%9C-%ED%83%80%EC%9E%85%EC%9D%BC%EA%B9%8C)
[Unsafe 'PropsWithChildren' Utility type in React TypeScript
](https://dev.to/maafaishal/unsafe-propswithchildren-utility-type-in-react-typescript-app-3bd3)준혁님이 고민하신 것처럼 Children이 옵셔널인 것이 안전하지않다고 생각들면 안전한 타입을 새롭게 설계하는 방법도 있다고 합니다! 
참고용으로 한번 읽어봐도 좋을 것 같슴다!